mods/ts/rules/world.yaml

```diff
@@ -198,47 +198,39 @@
 		Name: Nod
 		InternalName: nod
 		Description: Brotherhood of Nod\nThe Brotherhood is a religious cult centered around their leader Kane\nand the alien substance Tiberium. They utilize stealth technology\nand guerilla tactics to defeat those who oppose them.
-	ResourceType@Tiberium:
-		Type: Tiberium
-		Name: Tiberium
-		ResourceType: 1
-		Palette: greentiberium
-		Sequences: tib01, tib02, tib03, tib04, tib05, tib06, tib07, tib08, tib09, tib10, tib11, tib12
-		MaxDensity: 12
-		ValuePerUnit: 25
-		AllowedTerrainTypes: Clear, Rough, DirtRoad
-		AllowUnderActors: true
-		TerrainType: Tiberium
-	ResourceType@BlueTiberium:
-		Type: BlueTiberium
-		Name: Tiberium
-		ResourceType: 2
-		Palette: bluetiberium
-		Sequences: tib01, tib02, tib03, tib04, tib05, tib06, tib07, tib08, tib09, tib10, tib11, tib12
-		MaxDensity: 12
-		ValuePerUnit: 40
-		AllowedTerrainTypes: Clear, Rough, DirtRoad
-		AllowUnderActors: true
-		TerrainType: BlueTiberium
-	ResourceType@Veins:
-		Type: Veins
-		Name: Veins
-		ResourceType: 3
-		Palette: player
-		Sequences: veins
-		MaxDensity: 1
-		ValuePerUnit: 0
-		AllowedTerrainTypes: Clear, Rough, DirtRoad
-		AllowUnderActors: true
-		TerrainType: Veins
 	TerrainGeometryOverlay:
 	DebugVisualizations:
 	ExitsDebugOverlayManager:
 	CliffBackImpassabilityLayer:
 	SubterraneanActorLayer:
 	JumpjetActorLayer:
-	ResourceRenderer:
-		RenderTypes: Tiberium, BlueTiberium, Veins
+	TSTiberiumRenderer:
+		Resources:
```

Comment:
Nit: Wouldn't it be more consistent to call this `ResourceTypes`, since you use `ResourceType` on the VeinsRenderer (and internally)?

Reply:
My logic here was that `ResourceType` refers to the string label, whereas these definitions are listing the resource info for a given type which make them a bit different. I don't mind changing this if you or others still prefer `ResourceTypes`.

Reply:
OK, makes sense. I don't really have a preference here, so as far as I'm concerned we can leave it like that then.

Reply:
After more thought I think I will change this.